src/main/scala/piconot/RightHand.txt

```diff
@@ -0,0 +1,15 @@
+From State 0, if [E: open] -> go E, change to State 1
+From State 0, if [N: open, E: blocked] -> go N, change to State 0
+From State 0, if [N: blocked, E: blocked] -> go Nowhere, change to State 3
+
+From State 1, if [S: open] -> go S, change to State 3
+From State 1, if [E: open, S: blocked] -> go E, change to State 1
+From State 1, if [E: blocked, W: open, S: blocked] -> go Nowhere, change to State 2
+
+From State 2, if [N: open] -> go N, change to State 0
+From State 2, if [N: blocked, W: open] -> go W, change to State 2
+From State 2, if [N: blocked, E: open, W: blocked] -> go Nowhere, change to State 1
+
+From State 3, if [W: open] -> go W, change to State 2 
+From State 3, if [W: blocked, S: open] -> go S, change to State 3
+From State 3, if [N: open, W: blocked, S: blocked] -> go Nowhere, change to State 0
```

Comment:
Same comment as above:

How easy was it to run? This code and the following in RightHand.txt were both very easy to run!
Did the instructions work? This code and the following in RightHand.txt had instructions that worked great.

Modify the example programs to introduce errors. How robust is their implementation? What do you like about their error-handling? What can be improved?

The error handling for both of these files this and the RightHand.txt worked very well. The programs however are not very robust. I could not make changes to these files without getting a parsing error. the syntax of their language is very strict and needed to happen in the order of "From ... if [] -> go x ... change". I tried modifying the order of this and it did not work which is good! It would be cool if in future implementations if I could write these instructions in any order.